ansible/configs/ocp4-cluster/destroy_env_ec2.yml

```diff
@@ -170,5 +172,36 @@
     include_role:
       name: host-ocp4-destroy
 
+- name: Remove workloads
+  hosts: localhost
+  connection: local
+  gather_facts: false
+  run_once: true
+  become: false
+  tasks:
+  - name: Get openshift route53 A records to delete
+    # community.aws.route53:
+    mark_route53:
+      state: get
+      aws_access_key: "{{ aws_access_key_id }}"
+      aws_secret_key: "{{ aws_secret_access_key }}"
+      zone: "{{ guid }}{{ subdomain_base_suffix }}"
+      record: "{{ item }}"
+      type: "A"
+    with_items:
+      - "api.cluster-{{ guid }}.{{ guid }}{{ subdomain_base_suffix }}"
+      - "*.apps.cluster-{{ guid }}.{{ guid }}{{ subdomain_base_suffix }}"
+    register: records
+  - debug: var=records
+  - name: Delete openshift route53 A records
+    # community.aws.route53:
+    mark_route53:
+      state: absent
+      aws_access_key: "{{ aws_access_key_id }}"
+      aws_secret_key: "{{ aws_secret_access_key }}"
+      zone: "{{ item.set.zone }}"
+      record: "{{ item.set.record }}"
+      type: "{{ item.set.type }}"
+    with_items: "{{ records.results }}"
```

Comment:
did you add this part intentionally?

Reply:
yes. eventually another pr

Reply:
Current code does not remove these records and blocks cloudformation stack from deleting.